src/Behat/Behat/HelperContainer/Exception/NotFoundException.php

```diff
@@ -10,18 +10,12 @@
 
 namespace Behat\Behat\HelperContainer\Exception;
 
-class_alias(
-    interface_exists('Interop\\Container\\Exception\\NotFoundException')
-        ? 'Interop\\Container\\Exception\\NotFoundException'
-        : 'Psr\\Container\\NotFoundExceptionInterface',
-    'Behat\\Behat\\HelperContainer\\Exception\\NotFoundException'
```

Comment:
If we wanted to be *certain* that this wouldn't affect end users, we could update our composer.json to conflict with all versions of `container-interop` before 1.2.0.

That was the [final release of container-interop](https://github.com/container-interop/container-interop/pull/91) in Feb 2017, and in that release the interfaces we're aliasing just extend the PSR interfaces so this extra hop is redundant.

Equally I think it's very unlikely anyone still has an older version of container-interop installed at this point (1.1.0 was in January 2015).